meteor/client/lib/rundown.ts

```diff
@@ -386,6 +391,14 @@ export namespace RundownUtils {
 					hasAlreadyPlayed = true
 				}
 
+				const pieceInstanceFieldOptions: FindOptions<PartInstance> = {
+					fields: {
+						//@ts-ignore deep property
+						'piece.startedPlayback': 0,
```

Comment:
Good catch! What about the `startedPlayback` and `stoppedPlayback` properties on PieceInstance? I didn't notice them before on the type, but I think I would like to "field" them out of reactivity, because they aren't really all that important to the GUI and we generally have too much reactivity anyway.

Reply:
yeah, this filter should be updated to those fields instead